db/migrate/20240918153233_add_submission_type_to_made_live_forms.rb

```diff
@@ -0,0 +1,18 @@
+class AddSubmissionTypeToMadeLiveForms < ActiveRecord::Migration[7.1]
+  def change
+    reversible do |direction|
+      MadeLiveForm.find_each do |made_live_form|
```

Comment:
Whenever we use model code directly in our migrations we should stub the class ie ```class MadeLiveForm < ActiveRecord::Base
    self.table_name = :forms
  end``` to avoid any errors in re-running migrations say if validations changed or the model is removed altogether.

Reply:
Thanks! Have added this